diff --git a/vite.config.ts b/vite.config.ts
index 6620d4372213450ce8ab9a3b55bca727e7edefa5..d520bae5616a2166bcd37ed11d9f721eee1be536 100644
--- a/vite.config.ts
+++ b/vite.config.ts
@@ -15,6 +15,7 @@ export default defineConfig(async () => {
 
   return {
     root: packageRoot,
+    base: "status.seek.com",
     plugins: [react(), tailwindcss()],
     define: {
       __CONFIG_PATH__: JSON.stringify(configPath),
